.requirements

```diff
@@ -23,3 +23,38 @@ V8=12.0.267.17
+LIBEXPAT_SHA256=d4cf38d26e21a56654ffe4acd9cd5481164619626802328506a2869afab29ab3 # 2.6.2
+LUAROCKS_SHA256=25f56b3c7272fb35b869049371d649a1bbe668a56d24df0a66e3712e35dd44a6 # 3.11.0
+NFPM_LINUX_X86_64_SHA256=6dd3b07d4d6ee373baea5b5fca179ebf78dec38c9a55392bae34040e596e4de7 # 2.31.0
+NFPM_LINUX_ARM64_SHA256=e6487dca9d9e9b1781fe7fa0a3d844e70cf12d92f3b5fc0c4ff771aa776b05ca
+NFPM_DARWIN_X86_64_SHA256=19954ef8e6bfa0607efccd0a97452b6d571830665bd76a2f9957413f93f9d8cd
+NFPM_DARWIN_ARM64_SHA256=9fd82cda017cdfd49b010199a2eed966d0a645734d9a6bf932c4ef82c8c12c96
+OPENSSL_SHA256=83c7329fe52c850677d75e5d0b0ca245309b97e8ecbcfdc1dfdc4ab9fac35b39 # 3.2.1
+PCRE_SHA256=889d16be5abb8d05400b33c25e151638b8d4bac0e2d9c76e9d6923118ae8a34e # 10.43
+OPENRESTY_SHA256=32ec1a253a5a13250355a075fe65b7d63ec45c560bbe213350f0992a57cd79df # 1.25.3.1
+WASMX_WASMER_LINUX_X86_64_SHA256=3db46d2974b2c91aba2f0311dc26f59c1def473591768cddee8cdbf4783bf2c4 #3.1.1
+WASMX_WASMER_LINUX_ARM64_SHA256=c212eebdf1cc6bf71e7b56d7421eb3494c3a6ab1faf50a55150b7522183d1d36
+WASMX_WASMER_DARWIN_X86_64_SHA256=008610ddefdd3e04af9733969da616f9a344017db451476a1ee1cf6702895f02
+WASMX_WASMER_DARWIN_ARM64_SHA256=8534b278c1006ccc7f128bd1611636e12a33b9e625344331f9be3b56a5bb3286
+WASMX_V8_LINUX_X86_64_SHA256=06b617a2b90ef81c302421937691e4f353ce2a2f3234607a8d270b1196c410f2 # 12.0.267.17
+WASMX_V8_LINUX_ARM64_SHA256=1e086105c27e9254ac2731eaf3dfb83d3966caa870ae984f5c92284bd26d1a3c
+WASMX_V8_DARWIN_X86_64_SHA256=0ed81aae1336720aaec833c37aa6bb2db2b611e044746d65d497f285dff367ac
+WASMX_WASMTIME_LINUX_X86_64_SHA256=bb6859c30de292db22b32522e3ef42239cebef93f42c9cad2e0306311122e72a # 19.0.0
+WASMX_WASMTIME_LINUX_ARM64_SHA256=63b2bd25828cf882befe7a6e9d5162c9a750f0ab1dbc2160f778eea48e9b52f7
+WASMX_WASMTIME_DARWIN_X86_64_SHA256=2a79e92fb4150b9389d9ec67da0ba9ab913b9207122050a5e183a3695645692f
+WASMX_WASMTIME_DARWIN_ARM64_SHA256=118b36b69953f00cebd9b5901b3313a19dea58eea926236a7318309f053e27a0
+GH_LINUX_AMD64_SHA256=5aee45bd42a27f5be309373c326e45cbcc7f04591b1798581a3094af767225b7 # 2.30.0
+GH_LINUX_ARM64_SHA256=3ef741bcc1ae8bb975adb79a78e26ab7f18a246197f193aaa8cb5c3bdc373a3f
+GH_MACOS_AMD64_SHA256=6b91c446586935de0e9df82da58309b2d1b83061cfcd4cc173124270f1277ca7
+GH_MACOS_ARM64_SHA256=32a71652367f3cf664894456e4c4f655faa95964d71cc3a449fbf64bdce1fff1
+PROTOC_SHA256=2994b7256f7416b90ad831dbf76a27c0934386deb514587109f39141f2636f37 # 3.19.0
+ALPINE_GCC11_X86_64_SHA256=4fbc9a48f1f7ace6d2a19a1feeac1f69cf86ce8ece40b101e351d1f703b3560c # 0.4.0
+ALPINE_GCC11_ARM64_SHA256=abd7003fc4aa6d533c5aad97a5726040137f580026b1db78d3a8059a69c3d45b # 0.4.0
+RHEL9_GCC11_ARM64_SHA256=40fcf85e8315869621573512499aa3e2884283e0054dfefc2bad3bbf21b954c0 # 0.5.0
+RHEL8_GCC8_ARM64_SHA256=7a9a28ccab6d3b068ad49b2618276707e0a31b437ad010c8969ba8660ddf63fb # 0.5.0
+AWS2023_GCC11_ARM64_SHA256=01498b49c20255dd3d5da733fa5d60b5dad4b1cdd55e50552d8f2867f3d82e98 # 0.5.0
+AWS2_GCC7_ARM64_SHA256=9a8d0bb84c3eea7b662192bf44aaf33a76c9c68848a68a544a91ab90cd8cba60 # 0.5.0
```

Comment:
let's only move those are have version number in `.requirements` is enough

Reply:
the purpose of this ticket is to have a single place to change when developer bump a dependency. for example,
the toolchains have version set in the `.bzl` files instead of `.requirements`, so the checksum also stays in the `.bzl` file.

Reply:
Okay, I see, it has been updated.